src/problem-domain-analysis/classes-objects-and-structure.tex

```diff
@@ -5,20 +5,21 @@ \section{Classes, Objects and Structure}\label{sec:classes-objects-and-structure
 In this section the classes, objects and structure of the proposed solution will be explored, this
 will help get a more detail understanding of how the system should function in relation to the problem at hand.
 
-
 % textidote: ignore begin
 \subsection{Classes and Objects}\label{subsec:classes-and-objects}
 % textidote: ignore end
+
 To understand the classes that are relevant for the issues Nova café is facing,
 a general description of the classes is given in Table~\ref{tab:class-table}.
 
-% textidote: ignore begin
 \begin{table}[H]
     \centering
     \begin{tabular} { m{2.5cm} m{10cm} }
         \toprule
         \textbf{Classes} & \textbf{Description} \\
+% textidote: ignore begin
```

Comment:
You probably want to add `midrule` to the `--remove-macros` arguments in `.textidote` if it complains about incorrect grammar.
And remove the ignores here